plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/ui/CloudSdkRuntimeWizardFragment.java

```diff
@@ -146,7 +146,7 @@ public void widgetSelected(SelectionEvent event) {
       }
     });
 
-    File location = new CloudSdkProvider(null).getCloudSdkLocation();
+    File location = new CloudSdkProvider().getCloudSdk().getJavaAppEngineSdkPath().toFile();
```

Comment:
Because there is already a Path class imported, this was my shortcut to having less readable java.nio.file.Path. I'll change to Path, though.

@briandealwis because we're only using this once in this class, I didn't factor this to a static variable. Let me know if there is anything I'm missing.